### PR TITLE
Adds various small changes

### PR DESCRIPTION
**Pull request by Syntro42**
Changes include removing villager death messages from chat bridge, adding alias to 'online' command, making commands case-insensitive, and fixing the list command to work with capitalized letters in roles

### CODE REVIEW

**Comment by Syntro42** [comment]
Change to the chat bridge actually doesn’t send all relevant messages. I’ll fix in a bit

File: packages/endbot/src/endbot.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ export class Endbot
 
 		try {
 			await discord.run(
-				message.content.slice(this.prefix.length),
+				message.content.toLowerCase().slice(this.prefix.length),
```
Comment (samipourquoi) [inline packages/endbot/src/endbot.ts:43]: I feel like this could cause some troubles

**Review by samipourquoi — APPROVED** [review]

**Comment by samipourquoi** [comment]
I'm giving you write access to the repo also

File: packages/endbot/src/endbot.ts (hunk 1 above)
Comment (Syntro42) [reply packages/endbot/src/endbot.ts:43]: Yep, thinking about it, that actually won’t work with custom arguments. I’ll revert it

**Pull request merged**